src/SharpRaven.sln

```diff
@@ -45,32 +37,59 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{F44F1C47-D
 		..\docs\sentry-doc-config.json = ..\docs\sentry-doc-config.json
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpRaven.Nancy", "app\SharpRaven.Nancy\SharpRaven.Nancy.csproj", "{ABE22746-6EEB-4970-A608-C02BC3B8BDA3}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpRaven.Nancy.UnitTests", "tests\SharpRaven.Nancy.UnitTests\SharpRaven.Nancy.UnitTests.csproj", "{7FE52A2B-430C-4C6F-BEA9-0855AF973D0C}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpRaven.UnitTests", "tests\SharpRaven.UnitTests\SharpRaven.UnitTests.csproj", "{E1DBEBBF-9448-4D99-B378-2B8CF1629F31}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpRaven.WebTest", "tests\SharpRaven.WebTest\SharpRaven.WebTest.csproj", "{156621FC-2C48-4CDF-A368-9347BABE9089}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug 3.5|Any CPU = Debug 3.5|Any CPU
+		Debug 3.5|Mixed Platforms = Debug 3.5|Mixed Platforms
+		Debug 3.5|x86 = Debug 3.5|x86
 		Debug 4.0|Any CPU = Debug 4.0|Any CPU
 		Debug 4.0|Mixed Platforms = Debug 4.0|Mixed Platforms
 		Debug 4.0|x86 = Debug 4.0|x86
 		Debug 4.5|Any CPU = Debug 4.5|Any CPU
 		Debug 4.5|Mixed Platforms = Debug 4.5|Mixed Platforms
 		Debug 4.5|x86 = Debug 4.5|x86
+		Release 3.5|Any CPU = Release 3.5|Any CPU
+		Release 3.5|Mixed Platforms = Release 3.5|Mixed Platforms
+		Release 3.5|x86 = Release 3.5|x86
 		Release 4.0|Any CPU = Release 4.0|Any CPU
 		Release 4.0|Mixed Platforms = Release 4.0|Mixed Platforms
 		Release 4.0|x86 = Release 4.0|x86
 		Release 4.5|Any CPU = Release 4.5|Any CPU
 		Release 4.5|Mixed Platforms = Release 4.5|Mixed Platforms
 		Release 4.5|x86 = Release 4.5|x86
+		Release|Any CPU = Release|Any CPU
+		Release|Mixed Platforms = Release|Mixed Platforms
+		Release|x86 = Release|x86
```

Comment:
Which version of the .NET framework does the version-less build configurations build against? Should we have them?